cognitive/src/test/scala/com/microsoft/ml/spark/cognitive/split1/TranslatorSuite.scala

```diff
@@ -0,0 +1,405 @@
+
+  lazy val dictionaryLookup: DictionaryLookup = new DictionaryLookup()
+    .setSubscriptionKey(translatorKey)
+    .setLocation("eastus")
+    .setFromLanguage("en")
+    .setToLanguage("es")
+    .setTextCol("text")
+    .setOutputCol("result")
+
+  test("Break sentence ") {
+    val results = dictionaryLookup.transform(dictDf)
+      .withColumn("translations", flatten(col("result.translations")))
+      .withColumn("normalizedTarget", col("translations.normalizedTarget"))
+      .select("normalizedTarget").collect()
+    val headStr = results.head.getSeq(0).mkString("\n")
+    assert(headStr === "volar\nmosca\noperan\npilotar\nmoscas\nmarcha")
+  }
+
+  override def testObjects(): Seq[TestObject[DictionaryLookup]] =
+    Seq(new TestObject(dictionaryLookup, dictDf))
+
+  override def reader: MLReadable[_] = DictionaryLookup
+}
+
+class DictionaryExamplesSuite extends TransformerFuzzing[DictionaryExamples]
+  with TranslatorKey with Flaky with TranslatorUtils {
+
+  import spark.implicits._
+
+  lazy val dictDf: DataFrame = Seq(List(("fly", "volar"))).toDF("textAndTranslation")
+
+  lazy val dictionaryExamples: DictionaryExamples = new DictionaryExamples()
+    .setSubscriptionKey(translatorKey)
+    .setLocation("eastus")
+    .setFromLanguage("en")
+    .setToLanguage("es")
+    .setTextAndTranslationCol("textAndTranslation")
+    .setOutputCol("result")
+
+  test("Dictionary Examples") {
+    val results = dictionaryExamples.transform(dictDf)
+      .withColumn("examples", flatten(col("result.examples")))
+      .withColumn("sourceTerm", col("examples.sourceTerm"))
+      .withColumn("targetTerm", col("examples.targetTerm"))
+      .select("sourceTerm", "targetTerm").collect()
+    assert(results.head.getSeq(0).head.toString === "fly")
+    assert(results.head.getSeq(1).head.toString === "volar")
+  }
+
+  override def testObjects(): Seq[TestObject[DictionaryExamples]] =
+    Seq(new TestObject(dictionaryExamples, dictDf))
+
+  override def reader: MLReadable[_] = DictionaryExamples
+}
+
+class DocumentTranslatorSuite extends TransformerFuzzing[DocumentTranslator]
+  with TranslatorKey with Flaky {
+
+  import spark.implicits._
+
+  // TODO: Replace root SAS urls after 2022-07-07
+  lazy val sourceRoot: String = "?sp=rl&st=2021-07-06T06:28:26Z&se=2022-07-07T06:28:00Z" +
+    "&sv=2020-08-04&sr=c&sig=h9zzqvBdrvM81%2BWxuoG0bgNnn5lGbTaGcy27qyZDZm4%3D"
+
+  lazy val targetRoot: String = "?sp=racwl&st=2021-07-06T06:29:05Z&se=2022-07-07T06:29:00Z" +
+    "&sv=2020-08-04&sr=c&sig=tk62GpHoRb5CcojmyQammMbnYICAsTdgQqAeCikbtKg%3D"
+
+  lazy val sourceUrl: String = "https://mmlspark.blob.core.windows.net/datasets" + sourceRoot
+
+  lazy val fileSourceUrl: String = "https://mmlspark.blob.core.windows.net/datasets/Translator/" +
+    "source/document-translation-sample.pdf" + sourceRoot
+
+  lazy val targetUrl: String = "https://mmlspark.blob.core.windows.net/translator-target/test-zh-Hans-" +
```

Comment:
It looks like this is a new container as opposed to using the datasets container, are we unable to re-use datasets container for everything?